app/renderer/js/tray.ts

```diff
@@ -3,14 +3,13 @@ import {ipcRenderer, remote, WebviewTag, NativeImage} from 'electron';
 import path from 'path';
 import * as ConfigUtil from './utils/config-util';
 
-const {Tray, Menu, nativeImage, BrowserWindow, nativeTheme} = remote;
+const {Tray, Menu, nativeImage, BrowserWindow} = remote;
 
 let tray: Electron.Tray;
 
 // Get the theme on macOS
-const theme = nativeTheme.shouldUseDarkColors ? 'dark' : 'light';
 
-const ICON_DIR = process.platform === 'darwin' ? `../../resources/tray/${theme}` : '../../resources/tray';
+const ICON_DIR = process.platform === 'darwin' ? '../../resources/tray/macOS' : '../../resources/tray';
```

Comment:
There’s no longer a reason to have the macOS icon in a different directory, right? This could just always be `../../resources/tray`.

Reply:
I had the thought but preserved the extra directory so that the other contributors could comprehend what is Template Image and where it comes into play. (though we do have osx in the name)
If you feel I might shift it one dir up?

Reply:
Yeah, I don’t see the extra directory doing anything to improve comprehensibility. Use the same directory to simplify the code. Feel free to rename the files to `tray-macOS-Template.png`, etc. or something.